tests/imap_integration.rs

```diff
@@ -288,7 +343,7 @@ fn list() {
     let mut s = session("readonly-test@localhost");
     s.select("INBOX").unwrap();
     let subdirs = s.list(None, Some("%")).unwrap();
-    assert_eq!(subdirs.len(), 0);
+    assert_eq!(subdirs.len(), 1);
```

Comment:
This feels like a weird change. Why did the length suddenly change?

Reply:
Oh, right, I guess the test was ignored before so it was probably wrong.

Reply:
:);.  I only changed it because I currently have cyrus running all ignored tests.